substrate/test-utils/runtime/src/lib.rs

```diff
@@ -206,19 +238,26 @@ decl_runtime_apis! {
 		fn vec_with_capacity(size: u32) -> Vec<u8>;
 		/// Returns the initialized block number.
 		fn get_block_number() -> u64;
-
 		/// Test that `ed25519` crypto works in the runtime.
 		///
-		/// Returns the signature generated for the message `ed25519` and the public key.
-		fn test_ed25519_crypto() -> (ed25519::AppSignature, ed25519::AppPublic);
+		/// Returns the signature generated for the message `ed25519` public key and proof of possession.
```

Comment:
```suggestion
		/// Returns the signature generated for the message `ed25519` the public key and the proof of possession.
```
Out of curiosity why message has 'the' but not public key nor the proof of possession.